hazelcast/src/test/java/com/hazelcast/test/starter/constructor/AbstractConfigConstructor.java

```diff
@@ -63,6 +63,10 @@ static Object cloneConfig(Object thisConfigObject, ClassLoader classloader) thro
             return cloneSplitBrainProtectionFunctionImplementation(thisConfigObject, otherConfigClass);
         }
 
+        if (thisConfigClass.getName().equals("com.hazelcast.jet.config.InstanceConfigDelegate")) {
```

Comment:
Why do we use class names as string here? There are many instances in this class. Why don't we use this?:
```suggestion
        if (thisConfigClass.getName().equals(InstanceConfigDelegate.class.getName())) {
```

Reply:
I don't know the reasoning exactly but I used string because other checks in the class use string too.  I decided to leave it as is